### PR TITLE
 🎉 New Source: MailerLite

### PR DESCRIPTION
**Pull request by rytismi**
## What
The following PR adds the new low code CDK-based MaileLite Connector. For more context, please refer to [this issue](https://github.com/airbytehq/connector-contest/issues/128).

## How
This PR uses the lowcode CDK to create a MailerLite connector with the following streams:

- subscribers
- segments
- automations
- campaigns
- timezones
- forms_popup
- forms_embedded
- forms_promotion

## Recommended reading order
1.  `spec.yaml`
2. `mailerlite.yaml`
3. schemas

## 🚨 User Impact 🚨
No changes to existing code.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

## Tests

<details><summary><strong>Acceptance</strong></summary>

310443703_1171823047070066_896567056139172079_n

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-mailerlite/integration_tests/catalog.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
+    {
+      "name": "TODO fix this file",
+      "supported_sync_modes": ["full_refresh", "incremental"],
+      "source_defined_cursor": true,
+      "default_cursor_field": "column1",
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-mailerlite/integration_tests/catalog.json:7]: Remove this file.

File: airbyte-integrations/connectors/source-mailerlite/source_mailerlite/schemas/TODO.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# TODO: Define your stream schemas
+Your connector must describe the schema of each stream it can output using [JSONSchema](https://json-schema.org). 
+
+You can describe the schema of your streams using one `.json` file per stream.
+ 
+## Static schemas
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-mailerlite/source_mailerlite/schemas/TODO.md:6]: Remove this file.

File: airbyte-integrations/connectors/source-mailerlite/source_mailerlite/spec.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/mailerlite
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Mailerlite Spec
+  type: object
+  required:
+    - api_token
+  additionalProperties: true
+  properties:
+    # 'TODO: This schema defines the configuration required for the source. This usually involves metadata such as database and/or authentication information.':
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-mailerlite/source_mailerlite/spec.yaml:10]: Remove this comment.

File: docs/integrations/sources/mailerlite.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+### Performance considerations
+
+MailerLite API has a global rate limit of 120 requests per minute.
+
+## Getting started
+
+### Requirements
+
+* MailerLite API Key
```
Comment (marcosmarxm) [inline docs/integrations/sources/mailerlite.md:33]: Add the changelog info, check other connectors.

File: airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+      paginator:
+        $ref: "*ref(definitions.increment_paginator)"
+
+  # API Docs: https://developers.mailerlite.com/docs/segments.html
+  segments_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "segments"
+      primary_key: "id"
+      path: "/api/segments"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml:63]: Why `api` doesn't belongs to the base url?

File: airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml [hunk 6]
```diff
@@ -0,0 +1,155 @@
+version: "0.1.0"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml:1]: The implementation looks great and clean! Congrats @rytismi :)

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
A few comments, but probably ready to go after them. Do you have a integration test api key to share to our team run the tests too?

File: airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml (hunk 5 above)
Comment (dainiussa) [reply airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml:63]: Not sure but that's what their documentation says: https://developers.mailerlite.com/docs/#mailerlite-api

**Comment by dainiussa** [comment]
Addressed the requested changes :) We do have an API Key from the account we used for testing, how should we send it to you?

File: airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml (hunk 5 above)
Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-mailerlite/source_mailerlite/mailerlite.yaml:63]: ok, but you can simplify the path to the connectr.

**Comment by marcosmarxm** [comment]
> Addressed the requested changes :) We do have an API Key from the account we used for testing, how should we send it to you? @marcosmarxm

Can you share them with me? Send me a DM in Slack.

**Review by marcosmarxm — APPROVED** [review]
congrats @rytismi I'll publish the connector later.

**Pull request merged**